### ISSUE TITLE
A question ablout self.down 

### ISSUE THREAD

**Issue by buaacarzp** [ohhhyeahhh/SiamCAR#24]
我看到在将三个特征层concate后你们的通道数目是```(256*3)```，于是采用的下面的反卷积降低通道数：
```self.down = nn.ConvTranspose2d(256 * 3, 256, 1, 1)```
这里我想问的是为什么不直接用```1*1*256```的卷积核进行卷积而是用了反卷积，这点有没有做过相关的实验？

**Comment by WangJun-ZJUT** [comment]
Thank you for your question. In order to compare result between the ConvTranspose2d with Conv2d, we find that the kernel size is 768*256*1*1 when using nn.ConvTranspose2d and the kernel size of 256*768*1*1 is used in nn.Conv2d, so we transposed the kernel from 768*256*1*1 to 256*768*1*1 and convolution with the same input by nn.Conv2d, then we got the same results. However, it's a pity we didn't make a comparison in training.

**Comment by buaacarzp** [comment]
> Thank you for your question. In order to compare result between the ConvTranspose2d with Conv2d, we find that the kernel size is 768_256_1_1 when using nn.ConvTranspose2d and the kernel size of 256_768_1_1 is used in nn.Conv2d, so we transposed the kernel from 768_256_1_1 to 256_768_1_1 and convolution with the same input by nn.Conv2d, then we got the same results. However, it's a pity we didn't make a comparison in training.

请问你们想到将ResNet-50的 3,4,5层concate的动机是什么呢？有没有做过和siamrpn++一样的backbone得到一个结果呢？比如3，4，5层都接入一个head网络，再通过加权平均的方式计算最终的结果。

**Comment by WangJun-ZJUT** [comment]
It's a good question. An obvious advantage is that the number of parameters in head module can be reduced, besides the speed and accuracy can be better balanced by this way. Finally, more comparative experiments are welcomed to achieve, such as the one you mentioned above.

**Comment by buaacarzp** [comment]
thanks.
